hushline/__init__.py

```diff
@@ -28,7 +28,7 @@ def handle_rate_limit_exceeded(e):
 
     app.config["SECRET_KEY"] = os.getenv("SECRET_KEY")
 
-    app.config["SQLALCHEMY_DATABASE_URI"] = os.environ["SQLALCHEMY_DATABASE_URI"]
+    app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///hushline.db"
```

Comment:
This should still be set by the env var no matter what DB we use.

Reply:
Updated installer to remove MariaDB and include sqlite in the .env config